extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -128,6 +128,15 @@ const loadMetricHandlers = jsutil.once(repConfig => {
         buckets: repConfig.objectSizeMetrics,
     });
 
+    const maxReplayCount = !repConfig.replayTopics ? 0 : repConfig.replayTopics.reduce(
+        (prev, topic) => prev + topic.retries, 0);
+    const replayCount = ZenkoMetrics.createHistogram({
+        name: 'replication_replay_count',
+        help: 'Number of replay count to complete replication',
```

Comment:
```suggestion
        help: 'Number of replays to complete replication',
```
I think this is more grammatically correct